scripts/build.ps1

```diff
@@ -376,13 +376,19 @@ function Publish-Package
     # Copy the .NET core x86 and x64 testhost exes from tempPublish to required folder
     New-Item -ItemType directory -Path $testhostCore20PackageX64Dir -Force | Out-Null
     Copy-Item $testhostCore20PackageTempX64Dir\testhost* $testhostCore20PackageX64Dir -Force -recurse
+    Copy-Item $testhostCore20PackageTempX64Dir\Microsoft.TestPlatform.PlatformAbstractions.dll $testhostCore20PackageX64Dir -Force -recurse
```

Comment:
nit There is no need for -Recurse when you are copying just a single file, also you could fix the casing if you keep it -Recurse

Reply:
Just copied and pasted from the line above. Fixing.